setup.py

```diff
@@ -177,7 +177,7 @@ def write_version_py(filename="pyface/_version.py"):
             if len(c.split()) > 0
         ],
         description="traits-capable windowing framework",
-        long_description=open("README.rst").read(),
+        long_description=open("README.rst").read().split(".. end_of_long_description")[0],
```

Comment:
Not a request change because I know this is entirely orthogonal: It would also be nice to turn this into a context manager to ensure the opened file is closed as soon as we are done with it:
```
    with open("README.rst", "r", encoding="utf-8") as readme:
        return readme.read().split(".. end_of_long_description")[0]
```
Every little bit of drive-by modernization/improvements will add up. Then of course there is also a balance between keeping the PR focused and doing small improvements little by little... It is a judgment call that vary from individual to individual. :)

Reply:
I will merge now and then take a look at changing this later today.  Thank you!